app/resources/v1/pacbio/library_resource.rb

```diff
@@ -23,11 +27,11 @@ def self.records_for_populate(*_args)
       end
 
       def created_at
-        @model.created_at.to_s(:us)
+        @model.created_at.to_fs(:us)
```

Comment:
Any idea why we're calling our custom time format `:us`?
Introduced here: https://github.com/sanger/traction-service/pull/240/commits/e86453bc220535f8031da466b39a3a70163cd917

As part of this PR:
https://github.com/sanger/traction-service/pull/240